mix.lock

```diff
@@ -1,6 +1,6 @@
 %{
   "accent": {:hex, :accent, "1.1.1", "20257356446d45078b19b91608f74669b407b39af891ee3db9ee6824d1cae19d", [:mix], [{:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: true]}, {:plug, "~> 1.3", [hex: :plug, repo: "hexpm", optional: false]}], "hexpm", "6d5afa50d4886e3370e04fa501468cbaa6c4b5fe926f72ccfa844ad9e259adae"},
-  "beacon": {:git, "https://github.com/beaconCMS/beacon.git", "fdcade8bdd5a37fdb3601211e5c32c13ae50c202", []},
+  "beacon": {:git, "https://github.com/beaconCMS/beacon.git", "f520420c7dc673402e938323d2a54123b8c06875", []},
```

Comment:
@leandrocp This update now includes the stuff you added on https://github.com/BeaconCMS/beacon/pull/529